src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1429,6 +1441,21 @@ protected MembersAndInitializers GetMembersAndInitializers()
             return membersAndInitializers!;
         }
 
+        [Conditional("DEBUG")]
+        internal void AssertMemberExposure(Symbol member)
```

Comment:
Consider adding a comment here elaborating on what `Exposure` means (basically what invariant are we trying to maintain).